sharktank/sharktank/utils/cli.py

```diff
@@ -240,6 +240,21 @@ def add_export_artifacts(parser: argparse.ArgumentParser):
         help="Output file path for compiled vmfb file",
         type=str,
     )
+    parser.add_argument(
+        "--extra-compile-arg",
+        help=(
+            "Additional flag(s) to provide to the IREE compiler. "
+            "E.g. `--extra-compile-arg=--compile-mode=vm --extra-compile-arg=--iree-vm-target-extension-f32`"
+        ),
+        action="append",
+        type=str,
+        default=[],
+    )
+    parser.add_argument(
```

Comment:
Should have a default of False in order to keep current behaviour the same.

Reply:
It has `False` as default, through [argparse.BooleanOptionalAction](https://docs.python.org/3/library/argparse.html#argparse.BooleanOptionalAction).